pilot/pkg/networking/core/v1alpha3/httproute.go

```diff
@@ -181,6 +181,46 @@ func (configgen *ConfigGeneratorImpl) buildSidecarOutboundHTTPRouteConfig(node *
 	return out
 }
 
+// TODO: merge with IstioEgressListenerWrapper.selectVirtualServices
+func selectVirtualServices(virtualServices []config.Config, servicesByName map[host.Name]*model.Service) []config.Config {
```

Comment:
nit: Can you add a comment on what this does?

Reply:
yes, make sense